packages/api/tests/endpoints/providers/update-provider.js

```diff
@@ -113,7 +114,7 @@ test.serial('PUT updates an existing provider and returns it in listing', async
   const { results } = response.body;
   stub.restore();
   t.is(results.length, 1);
-  t.deepEqual(results[0], updatedProvider);
+  t.deepEqual(results[0].globalConnectionLimit, updateParams.globalConnectionLimit);
```

Comment:
Why did this test expectation change?

Reply:
Because the deep equals no longer works because the result comes back with the updatedAt and createdAt. 🤷‍♀ 
Anyway the globalConnectionLimit is the only thing ti changes.